### PR TITLE
RCAL-409 Additional changes to Roman RTD content.

### PR DESCRIPTION
**Pull request by mairanteodoro**
Resolves [RCAL-409](https://jira.stsci.edu/browse/RCAL-409)

This PR addresses some of the suggested changes to the Roman RTD content. This commit contain the changes that were deemed priority for the next release whereas the rest of the suggestions will be addresses in future PRs.

**Checklist**
- [x] added entry in `CHANGES.rst` under the corresponding subsection
- [ ] updated relevant tests
- [X] updated relevant documentation
- [x] updated relevant milestone(s)
- [x] added relevant label(s)

### CODE REVIEW

**Review by ddavis-stsci — APPROVED** [review]
Looks good. My only question is do we really need the developer install instructions on RTD? I'll defer to Nadia on this point.

**Comment by mairanteodoro** [comment]
> Looks good. My only question is do we really need the developer install instructions on RTD? I'll defer to Nadia on this point.

@ddavis-stsci I think it could go to that section focused on technical details about the pipeline that we intend to create.

**Comment by nden** [comment]
I think it's OK for the developer install instructions to be on RTD. It will be more important when we have users, during commissioning and later, since the latest improvements will be released on a set schedule and users will want to use the repository in some cases. Also, since this is an open source package we may get external contributions as we do with jwst.

**Comment by nden** [comment]
How about something like this for intro, please feel free to improve the wording.

```
Welcome to the documentation for `romancal`. This package contains the calibration pipeline for the WFI instrument on the Nancy Grace Roman telescope, which processes data by applying various corrections to produce science-ready, calibrated output products including fully calibrated individual exposures as well as high-level data products (mosaics, catalogs, etc.). The tools in this package allow users to run and configure the pipeline to custom process their Roman data. Additionally, the romancal package contains the interface to the Roman datamodels, the recommended method of reading and writing Roman data files in Python.
```

File: docs/roman/assign_wcs/main.rst [hunk 1]
```diff
@@ -3,9 +3,11 @@ Description
 ===========
 
 ``romancal.assign_wcs`` is the first step run on an image, after ``romancal.ramp_fitting``.
-It associates a World Coodrinate System (WCS) object with each science exposure. The WCS
-object transforms positions in the ``detector`` frame to positions in the ``world``
-coordinate frame - ICRS. The WCS can be accessed as an attribute of the ``meta`` object
+It associates a World Coodrinate System (WCS) object with each science exposure.
```
Comment (schlafly) [inline docs/roman/assign_wcs/main.rst:6]: Coodrinate -> Coordinate

**Review by schlafly — APPROVED** [review]

**Comment by schlafly** [comment]
Looks good to me.  Two cents: at least I'll benefit from having the developer install instructions, so I'd vote to keep them.

**Comment by nden** [comment]
@mairanteodoro Can you make the changes to the introduction and then it's ready to be merged.

**Review by nden — APPROVED** [review]

**Comment by mairanteodoro** [comment]
Resolved conflicts and recreated branch.

**Pull request closed without merging**